bookkeeper-dist/src/assemble/bkctl.xml

```diff
@@ -0,0 +1,116 @@
+<!--
```

Comment:
`bin-all` and `bin-server` are two server distributions for "server binary" distributions. so that's why it was prefixed with `bin` (in the context, there was only one server binary distribution, the file was called `bin`).

calling `bkctl` is to differentiate the package from others.